### PR TITLE
👷 Add extension usage tips doc and link in ui

### PR DESCRIPTION
**Pull request by cy-moi**
## Motivation
Help users to navigate using the extension.

## Changes
Added usage tips in extension readme file and direct users to it in the extension.
Screenshot 2024-05-24 at 13 23 27

## Testing

- [x] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: developer-extension/README.md [hunk 1]
```diff
@@ -40,3 +40,58 @@ Then, in Google Chrome:
...
+**⚠️Don’t forget to reset everything in the Info Tab after experimenting.**
+
+Info tab contains information about Session and RUM SDK configurations
+
+- **RUM/LOGS Configuration**: edit configuration files on the fly. When configuration changes apply, the extension will automatically reload the page. But for some configurations you might want to click on End Current Session to ensure that the changes kicked in.
+- **End current session**: manually end the current session within the extension. This will also end the current replay session.
+
+### Setting Tab
+
+**⚠️Don’t forget to reset everything in the Setting Tab after experimenting.**
```
Comment (BenoitZugmeyer) [inline developer-extension/README.md:81]: **💬 suggestion:** ‏Github flavored markdown has a special syntax for emphasis block: 
```suggestion
> [!IMPORTANT]  
> Don’t forget to reset everything in the Setting Tab after experimenting.
```
Renders as:
> [!IMPORTANT]  
> Don’t forget to reset everything in the Setting Tab after experimenting.

See: https://github.com/orgs/community/discussions/16925

File: developer-extension/README.md [hunk 2]
```diff
@@ -40,3 +40,58 @@ Then, in Google Chrome:
...
+
+#### Event Columns
+
+The Events List offers an interactive experience to visualize RUM events:
+
+- Drag and drop to reorder columns in the event list
+- Remove (by clicking on `x` in the column title) or add new columns:
+  - Add a new column from searching for a field by clicking on the `+column` icon at the right side of the header row.
+  - Add a new column from values in existing columns by right clicking on any attribute in the event json.
+- Copy queries and objects from the list by right clicking on any cell
```
Comment (BenoitZugmeyer) [inline developer-extension/README.md:68]: **💬 suggestion:** ‏it's not a right click
```suggestion
- Copy queries and objects from the list by clicking on any cell
```

File: developer-extension/README.md [hunk 3]
```diff
@@ -40,3 +40,58 @@ Then, in Google Chrome:
...
+
+## Usage Tips
+
+### Event Tab
+
+The Event Tab contains a list of events sent by the SDK and a menu of event types for quick filtering.
+
+#### Search Syntax
+
+We support a basic `key:value` search syntax which means you can search within the limits of [RUM event structures](https://docs.datadoghq.com/real_user_monitoring/explorer/search/), such as `action.target.name:action_name`. While all the fields can be added as columns in the event list, it does not mean the existing ones are valid fields: the `Description` summary is not part of RUM event structure. You can click on any `Description` cell for details of the event to look for correct key to search with.
```
Comment (BenoitZugmeyer) [inline developer-extension/README.md:52]: **💬 suggestion:** ‏don't talk about the Description column in the "Search Syntax" section, as "columns" are different than "search", and it's not quite related to the syntax. I'm not sure if we should mention it at all.

**Comment by jhgilbert** [comment]
Hi, I created a [docs review ticket](https://datadoghq.atlassian.net/jira/software/projects/DOCS/boards/199?selectedIssue=DOCS-8070) so we can follow up with an editorial pass when this is ready. Thank you!

File: developer-extension/src/panel/components/panel.tsx [hunk 4]
```diff
@@ -64,6 +64,18 @@ export function Panel() {
...
+          value={activeTab ?? PanelTabs.Events}
+          rightSection={
+            <Anchor
+              href="https://github.com/DataDog/browser-sdk/tree/main/developer-extension#browser-sdk-developer-extension"
+              target="_blank"
+            >
+              🔗 Docs
+            </Anchor>
+          }
+        ></Tabs.Tab>
```
[the remark this answers is not in the document]
Comment (thomas-lebeau) [reply developer-extension/src/panel/components/panel.tsx:78]: **💭 thought:** ‏ for a later hackathon, we could render the markdown doc in the extension itself.

[the remark this answers is not in the document]
Comment (cy-moi) [reply developer-extension/src/panel/components/panel.tsx]: I've updated the doc link to the right end of the tab list.

**Review by BenoitZugmeyer — APPROVED** [review]

File: developer-extension/src/panel/components/panel.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (BenoitZugmeyer) [reply developer-extension/src/panel/components/panel.tsx]: Or we could host that documentation in https://docs.datadoghq.com/, why not? One place to host all doc :)

**Review by jhgilbert — APPROVED** [review]
Approved with some suggestions, all just for consistency with the way we would edit the copy in the Datadog docs. Thank you!

**Pull request merged**